### PR TITLE
Add layout and some content

### PR DESCRIPTION
**Pull request by welschoff**
localhost_3000_

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+import { createCardComponent } from './lib/createCardComponent.js';
 
-document.querySelector('#app').innerHTML = `
-  <h1>Hello Vite!</h1>
-  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
-`;
+function renderApp() {
+  const appElement = document.querySelector('#app');
+
+  const cardComponent = createCardComponent();
```
Comment (AnkePfei2fer) [inline main.js:8]: I would put this line after you create the mainElement

**Review by fabiglahn — COMMENTED** [review]
Good job, Hendrik!

**Review by AnkePfei2fer — COMMENTED** [review]
lgtm!

File: main.js (hunk 1 above)
Comment (welschoff) [reply main.js:8]: thanks :)

File: style.css [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+}
+
+main {
+  background-color: var(--color-background-main);
+  color: var(--color-main-text);
+  padding: 2rem 2rem;
+}
+
+.card {
+  background-color: white;
```
Comment (DennisTarik) [inline style.css:54]: I would set this color as a variable. If you are going to change the theme the background will stay the same.

**Review by DennisTarik — COMMENTED** [review]
lgtm babe <3

File: style.css [hunk 3]
```diff
@@ -1,8 +1,57 @@
...
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: var(--font-family-headline);
+}
+
+header {
```
Comment (lmachens) [inline style.css:40]: Be careful using element selectors here. You might want to use `header` in other components too.

File: style.css [hunk 4]
```diff
@@ -1,8 +1,57 @@
...
   text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+  padding: 4rem 2rem;
+}
+
+main {
+  background-color: var(--color-background-main);
+  color: var(--color-main-text);
+  padding: 2rem 2rem;
```
Comment (lmachens) [inline style.css]:
```suggestion
  padding: 2rem;
```

**Review by lmachens — APPROVED** [review]

**Pull request merged**